arctic/_config.py

```diff
@@ -102,3 +102,8 @@ class FwPointersCfg(Enum):
 # play well with the bson encoder or if you always want your data in the data collection.
 # -------------------------------
 SKIP_BSON_ENCODE_PICKLE_STORE = bool(os.environ.get('SKIP_BSON_ENCODE_PICKLE_STORE'))
+
+# -------------------------------
+# Maximum size to bson encode.
```

Comment:
Can you add a longer description Maybe something like "Maximum size up to which the input will be bson_encoded and stored in the version doc instead of being pickled in the version store"

Reply:
Ok